src/drem/transform/cso_sa.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import geopandas as gpd
+from drem.filepaths import RAW_DIR
+from geodir.py import df_output
```

Comment:
```suggestion
```

Reply:
> 

Can you avoid importing dataframes across files and instead explicitely via `pd.read_parquet` instead.  Importing dataframes from other scripts tends to lead to dependency hell

Reply:
Yeah this was one of my more ridiculous ones...